app/migrations/Version20231207235400.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+class Version20231207235400 extends AbstractMauticMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable($this->prefix.'forms');
+        if ($table->hasColumn('lang')) {
+            throw new SkipMigration('Schema includes this migration');
+        }
```

Comment:
Is there any reason this isn't in the `preUp()` method like in other migrations?

Reply:
No reason. Fixed